packages/@vuepress/cli/index.js

```diff
@@ -23,39 +23,40 @@ if (!semver.satisfies(process.version, requiredVersion)) {
   process.exit(1)
 }
 
-const program = require('commander')
+const cli = require('cac')()
 
-exports.program = program
+exports.cli = cli
 exports.bootstrap = function ({
   plugins,
   theme
 } = {}) {
   const { path, logger, env } = require('@vuepress/shared-utils')
   const { dev, build, eject } = require('@vuepress/core')
 
-  program
+  cli
     .version(pkg.version)
-    .usage('<command> [options]')
-
-  program
-    .command('dev [targetDir]')
-    .description('start development server')
-    .option('-p, --port <port>', 'use specified port (default: 8080)')
-    .option('-h, --host <host>', 'use specified host (default: 0.0.0.0)')
-    .option('-t, --temp <temp>', 'set the directory of the temporary file')
-    .option('-c, --cache <cache>', 'set the directory of cache')
+    .help()
+
+  cli
+    .command('dev [targetDir]', 'start development server')
+    .option('-p, --port [port]', 'use specified port (default: 8080)')
+    .option('-h, --host [host]', 'use specified host (default: 0.0.0.0)')
```

Comment:
You should use `<>` if the flag requires a string-type value. i.e. it would fail on `--host` but work on `--host localhost`

Reply:
Thanks, updated at https://github.com/vuejs/vuepress/pull/1049/commits/9b99ca7de0c2760b8a1fa82984722a59713dde31